extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -570,4 +571,9 @@ export const connectionProfile = localize('connectionProfile', 'Select a connect
 export const userConnectionString = localize('userConnectionString', 'Enter connection string');
 export const selectConnectionString = localize('selectConnectionString', 'Select SQL connection string method');
 export const selectConnectionError = (err?: any) => err ? localize('selectConnectionError', "Failed to set connection string app setting: {0}", utils.getErrorMessage(err)) : localize('unableToSetConnectionString', "Failed to set connection string app setting");
-
+export const includePassword = localize('includePassword', 'Do you want to include the password from this connection in the local.settings.json?');
```

Comment:
`Do you want to include the password from this connection in your local.settings.json file?`